project/profiles/capacity4more/modules/c4m/og/c4m_og/plugins/entityreference/selection/C4MOgSelectionHandler.class.php

```diff
@@ -140,7 +140,7 @@ public function buildEntityFieldQuery(
       $target_access = og_user_access($group_type, $group['gid'], "create $node_type content");
       // Any member can edit a wiki page unless a power user has changed it
       // specifically for a specific node.
-      if (!empty($this->entity->nid) && $node_type == 'wiki_page') {
```

Comment:
This commit doesn't have the `!empty($this->entity->nid)` condition.
`empty($this->entity->nid)` will return TRUE when creating node, so
`!empty($this->entity->nid)` will always return FALSE, if we create a node.